### PR TITLE
Update GPU add-on testing for new SKU

### PR DESCRIPTION
**Pull request by lugi0**
Signed-off-by: Luca Giorgi <lgiorgi@redhat.com>

### CODE REVIEW

**Review by pablofelix — DISMISSED** [review]
lgtm

**Comment by lugi0** [comment]
RHODS was behaving strangely, launching again: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1019/

**Comment by lugi0** [comment]
RHODS seems completely borked on that cluster. I will test the PR again tomorrow.

File: tests/Resources/OCP.resource [hunk 1]
```diff
@@ -38,3 +38,13 @@ Verify Namespace Status
     @{NS_obj} =  OpenShiftCLI.Get  kind=Namespace  label_selector=${label}
     &{NS_obj_dictionary} =  Set Variable  ${NS_obj}[0]
     Should Be Equal  ${NS_obj_dictionary.status.phase}  Active
+
+Verify CR Status
+    [Documentation]  Verifies that a CR exists, that it's available
+    ...    and that the name matches
+    [Arguments]  ${CRD}  ${CR_name}
+    @{CR_obj} =  OpenShiftCLI.Get  kind=${CRD}
+    &{CR_obj_dictionary} =  Set Variable  ${CR_obj}[0]
```
Comment (tarukumar) [inline tests/Resources/OCP.resource:47]: As per our style guide we should not have name with Capitals letter for limited scope variable. Could you please change it

Comment (tarukumar) [reply tests/Resources/OCP.resource:47]: https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit#heading=h.4a2fb94fltk2

**Comment by lugi0** [comment]
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1035/console
PR test, all passed

**Review by bdattoma — DISMISSED** [review]

File: tests/Resources/OCP.resource (hunk 1 above)
Comment (lugi0) [reply tests/Resources/OCP.resource:47]: I've added a draft, look it over when you have some time @bdattoma @tarukumar @jgarciao

Comment (jgarciao) [reply tests/Resources/OCP.resource:47]: I've added my toughs to the document

Comment (jgarciao) [reply tests/Resources/OCP.resource:47]: Agreement: we'll use lowercase for local keywords and parameters

**Review by jgarciao — APPROVED** [review]

**Review by bdattoma — APPROVED** [review]

**Pull request merged**